### PR TITLE
Milestone5

### PR DESCRIPTION
**Pull request by azdrenymeri**
### Hello TSE-s 👋 
* I had this small issue
scr
I had to set the `total` property to `0`  otherwise it was throwing this error.
scr
#### (BTW: the design of the calculator is done correctly as requested on milestone3 with the right colors)

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -19,8 +22,10 @@ const operate = (numberOne, numberTwo, operator) => {
             res = a.div(b);
```
Comment (SunnyAsar) [inline src/logic/operate.js]: Hi, we need to handle the  `case '÷':`
dividing by zero breaks the app. instead lets set error message like the calculator does.

File: src/logic/calculate.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ import operate from './operate';
 
 const calculate = (calculatorObj, buttonName) => {
     const{total, next, operation} = calculatorObj;
```
Comment (SunnyAsar) [inline src/logic/calculate.js]: we also have a problem with behaviour. Here is the scenario:
```
5 + 5 = 10
then:
5 + 2 = 12
and not 7
```
instead the last total is used as first value and takes the operation `+` and number `2` for another calculation.

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -1,44 +1,42 @@
...
+                        <Buton name="÷"  wide={false}  clickHandler={props.clickHandler} />                        
                     </div>
                     <div className="ButtonPanel-Group">
-                        <Buton name="7" color="#e0e0e0" wide={false}/>
-                        <Buton name="8" color="#e0e0e0" wide={false}/>
-                        <Buton name="9" color="#e0e0e0" wide={false}/>
+                        <Buton name="7" color="#e0e0e0" wide={false} clickHandler={props.clickHandler} />
+                        <Buton name="8" color="#e0e0e0" wide={false} clickHandler={props.clickHandler} />
+                        <Buton name="9" color="#e0e0e0" wide={false} clickHandler={props.clickHandler}/>
                         <Buton name="x"  wide={false}/>
```
Comment (SunnyAsar) [inline src/components/ButtonPanel.js]: The click handler for multiply is missing, so no multiplication so far.

File: src/logic/calculate.js [hunk 4]
```diff
@@ -40,12 +39,13 @@ const calculate = (calculatorObj, buttonName) => {
...
             break;
         default:
-            if(next){
+            if(next !== '0' && next){
                 calculatorObj.next += buttonName;
             } else {
                 calculatorObj.next = buttonName;
             }
     }
+    console.log(calculatorObj)
```
Comment (SunnyAsar) [inline src/logic/calculate.js]: please remove log messages, the one in operate too.

File: src/components/ButtonPanel.js (hunk 3 above)
Comment (azdrenymeri) [reply src/components/ButtonPanel.js]: Done

File: src/logic/calculate.js (hunk 2 above)
Comment (azdrenymeri) [reply src/logic/calculate.js]: ### Fixed  🤓

File: src/logic/operate.js (hunk 1 above)
Comment (azdrenymeri) [reply src/logic/operate.js]: Fixed

File: src/logic/calculate.js (hunk 4 above)
Comment (azdrenymeri) [reply src/logic/calculate.js]: Done

**Pull request merged**